src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -248,6 +250,41 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
             "VBS")]
         public string PreferredEnclaveType { get; set; }
 
+        /// <summary>
+        /// Switch parameter to control if database identity is to be assigned.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Generate and assign an Azure Active Directory Identity for this database for use with key management services like Azure KeyVault.")]
+        public SwitchParameter AssignIdentity { get; set; }
+
+        /// <summary>
+        /// Database encryption protector
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The encryption protector key for SQL Database.")]
+        public string EncryptionProtector { get; set; }
+
+        /// <summary>
+        /// List of user assigned managed identities
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The list of user assigned identity for the SQL Database.")]
+        public List<string> UserAssignedIdentityId { get; set; }
```

Comment:
Please use string array to type of UserAssignedIdentityId, see https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#array-vs-enumerable-types